website/docs/guides/core-to-cloud-1.md

```diff
@@ -0,0 +1,244 @@
+---
+title: Move from dbt Core to dbt Cloud - Get started
+id: core-to-cloud-1
+description: "Learn how to move from dbt Core to dbt Cloud and what you need to get started."
+hoverSnippet: "Learn how to move from dbt Core to dbt Cloud."
+icon: 'guides'
+time_to_complete: 'Total estimated time: 3-4 hours'
+hide_table_of_contents: true
+tags: ['Migration','dbt Core','dbt Cloud']
+level: 'Intermediate'
+recently_updated: true
+---
+## Introduction
+
+Moving from dbt Core to dbt Cloud streamlines analytics engineering workflows by allowing teams to develop, test, deploy, and explore data products using a single, fully managed service.
+
+dbt Cloud is the fastest and most reliable way to deploy dbt. Develop, test, deploy, and explore data products using a single, fully managed service. It also supports: 
+  - The [dbt Cloud IDE](/docs/cloud/dbt-cloud-ide/develop-in-the-cloud) or command line with [dbt Cloud CLI](/docs/cloud/cloud-cli-installation) for development
+  - [dbt Semantic Layer](/docs/use-dbt-semantic-layer/dbt-sl) for consistent metrics
+  - Domain ownership of data with multi-project [dbt Mesh](/best-practices/how-we-mesh/mesh-1-intro) setups
+  - [dbt Explorer](/docs/collaborate/explore-projects) for easier data discovery and understanding
+  
+  Learn more about [dbt Cloud features](/docs/cloud/about-cloud/dbt-cloud-features).
+
+dbt Core is an open-source tool that enables data teams to transform data following analytics engineering best practices using a command line interface. It must be self-hosted and maintained.
+
+### What you'll learn
+
+This guide outlines what you need to do in order to move from dbt Core to dbt Cloud and highlights the necessary technical changes:
+
+- [Account setup](https://docs.getdbt.com/guides/core-to-cloud-1?step=3#account-setup): Learn how to create a dbt Cloud account, invite team members, and configure it for your team.
```

Comment:
It's not a "need to have," but for a guide like this, it can be beneficial to the reader to be direct. Instead of "Discover" and "Learn how," tell the reader what they'll be doing:
"Manage environment variables in dbt Cloud"
"Connect your data platform to dbt Cloud"
They learn in our docs pages, the guides are about doing.